samples/debug/gdbstub/prj.conf

```diff
@@ -0,0 +1,5 @@
+# nothing here
+CONFIG_GDBSTUB=y
+CONFIG_GDBSTUB_SERIAL_BACKEND_NAME="UART_1"
+CONFIG_NO_OPTIMIZATIONS=y
+CONFIG_USERSPACE=y
```

Comment:
why do we depend on userspace here?

Reply:
I found an issue when setting break points from user thread cause by a general fault exception due CPL value greater than DPL, so I decided keeping user thread.

Reply:
this sounds like something we should just fix, did you adjust the DPL in the IDT entry to allow ring 3 to trigger these exceptions?

Reply:
yep, added a new commit doing it